### ISSUE TITLE
twisted.trial.test.test_tests doesn't follow the coding standard

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6051]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#6051|
|Type|defect|
|Created|2012-10-04 13:21:43Z|

`twisted.trial.test.test_tests` should follow the coding standard. All tests should have docstrings, their name should start with `test_`, etc..

Attachments:

* [ticket-6051-docs.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/7f7/7f7aebea4d94cd047a68575a1b7db59904ec1eaa/a93a027bf137d41640633c566fdcc7e5907671ab.patch) (29941 bytes) - added by wirehead on 2013-03-18 20:36:03Z - patch to make twisted/trial/test/test_tests.py comply to standard.

<details><summary>Searchable metadata</summary>

```
trac-id__6051 6051
type__defect defect
reporter__itamar itamar
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__trial trial
keywords__None None
time__1349356903000000 1349356903000000
changetime__1474198049187875 1474198049187875
version__None None
owner__wsanchez wsanchez
cc__jml
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="weiteng's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|weiteng set owner to weiteng|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="weiteng's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|weiteng removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="wirehead's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|wirehead set owner to wirehead<br>wirehead set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="wirehead's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|wirehead commented|
|-|-|

Present style dings:
 * Docstrings:  
  * I am assuming that all those "See Module Docstrings" are not OK.
  * Some functions are missing docstrings altogether.
 * Naming
  * Some functions named testBlah instead of test_blah

**Comment by twisted-trac** [comment]
|<img alt="wirehead's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_6"></a>|wirehead removed owner<br>wirehead set status to `new`|
|-|-|

the ClassTest in GCMixin seems to be unnecessary and doesn't break anything.  Am I right about this?

Should be nothing but formatting and comments otherwise..

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_7"></a>|@wsanchez commented|
|-|-|

Very nice. +1.

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)|@wsanchez set owner to @wsanchez<br>@wsanchez set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_9"></a>|@wsanchez set status to `closed`|
|-|-|

(In [37638]) Apply ticket-6051-docs.patch: Add docs to test_tests.

Author: wirehead
Reviewer: wsanchez
Fixes: [#6051](https://github.com/twisted/twisted/issues/6051)

Make twisted/trial/test/test_tests.py comply to standard.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_10"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
